dictionary.txt

```diff
@@ -885,3 +885,5 @@ _rp	₎
 -	−	
 \taurus	♉	
 \cancer	♋	
+\male   ♂  
+\female ♀   
```

Comment:
Make sure to use tabs instead of spaces, the file won't work otherwise

```suggestion
\male	♂	
\female	♀	
```

Reply:
using tab only...not working for me...any specific text editor ...please

Reply:
You can try to copy paste the tab symbol from another line instead of pressing the tab key